dist/stat_data.py

```diff
@@ -339,8 +339,9 @@ def __init__(self, name, desc, flags=''):
     # Chunk cache statistics
     ##########################################
     ChunkCacheStat('chunk_cache_bytes_inuse', 'total bytes used by the cache'),
-    ChunkCacheStat('chunk_cache_chunks_inuse', 'total chunks held by the chunk cache'),
     ChunkCacheStat('chunk_cache_chunks_evicted', 'chunks evicted'),
+    ChunkCacheStat('chunk_cache_chunks_inuse', 'total chunks held by the chunk cache'),
+    ChunkCacheStat('chunk_cache_chunks_pinned', 'total number of pinned chunks cached in the chunk cache'),
```

Comment:
If we are going to stick with this, I would change the wording to make it as similar as possible to the "regular" in-use count - say, "total pinnned chunks held by the chunk cache".

Reply:
Agreed!

Reply:
Updated in [efc862d](https://github.com/wiredtiger/wiredtiger/pull/9480/commits/efc862d902141e082a164839bb362d0ea06e5503)